Sources/ProjectDescription/Config.swift

```diff
@@ -27,38 +27,23 @@
 ///     swiftVersion: "5.9.0"
 /// )
 /// ```
-public struct Config: Codable, Equatable, Sendable {
-    /// Generation options.
-    public let generationOptions: GenerationOptions
+public typealias Config = Tuist
 
-    /// Install options.
-    public let installOptions: InstallOptions
-
-    /// Set the versions of Xcode that the project is compatible with.
-    public let compatibleXcodeVersions: CompatibleXcodeVersions
-
-    /// List of `Plugin`s used to extend Tuist.
-    public let plugins: [PluginLocation]
-
-    /// Cloud configuration.
-    public let cloud: Cloud?
+public struct Tuist: Codable, Equatable, Sendable {
```

Comment:
nit: Let's move this definition to a new file `Sources/ProjectDescription/Tuist.swift`